packages/sdks/web-js-sdk/package.json

```diff
@@ -92,6 +92,9 @@
     "jwt-decode": "4.0.0",
     "tslib": "2.6.3"
   },
+  "peerDependencies": {
```

Comment:
Did you check the scenario where someone does not want to use it, and does not add it to the app dependencies?
I think it will break